### ISSUE TITLE
Current project status

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [OneGraph/graphiql-explorer#84]
@sgrove @dwwoelfel

Is this project still maintained?

I see many pull requests (e.g. #49, #60, etc.) for some reason are still not merged.

What the future of this project after OneGraph was acquired by Netlify?

Will this project be refactored from one-single file to component files? Will it be migrated from Flow to TypeScript? Will it support theme customization?

Will this project be developed as a part of GraphiQL 2?

**Comment by aggarwalnisha** [comment]
Hi @sgrove @dwwoelfel,

This is a really helpful tool.
Would love to know its current status.

We also see that there are some TODOs like supporting list arguments. Are there any plans to achieve them, or PRs are accepted for it?
Single file to component files and Flow to Typescript will also be great to do.

Also, can we pass props to disable multiple queries and running mutations?

**Comment by jonathanawesome** [comment]
@FluorescentHallucinogen @aggarwalnisha We're working on GraphiQL v2 over [here](https://github.com/graphql/graphiql/issues/2328), including an Explorer-inspired GUI query builder. Join us on [Discord](https://discord.com/channels/625400653321076807/966768858402816020)!

**Comment by tendant** [comment]
It will be great to have new release with updated dependencies, even there is no new feature or bug fixes. 

At least it can still be used by other projects. 

Even with the release of v2, there will be a lot projects depending on v1 for a while.

**Comment by haase1020** [comment]
@sgrove @dwwoelfel @jonathanawesome , I would also like to check on the status of this project. I would love to help if I can! I have a TypeScript version of the `Explorer.js` file, and would love to update dependencies and the code base. For example, you currently cannot upgrade to graphql v16 and without breaking the `isDeprecated` logic. Are there plans to make updates to this package? Is there any way to help?

**Comment by acao** [comment]
Here is my proposed future home of graphiql explorer:

- convert to typescript
- refactored into multiple files

https://github.com/graphql/graphiql/pull/3375

many more improvements to come, this is just a start
one major caveat is that the git history is lost with this approach. but it would make it much easier for us to maintain

**Comment by klippx** [comment]
@acao The PR you linked to was merged to `main`. But what happened after that? It seems to have been removed since... Did it not make sense to absorb this abandonded project into the graphiql monorepo?

**Comment by acao** [comment]
it was merged into [this branch](https://github.com/graphql/graphiql/pull/3374) so it hasn't been merged to main yet. we were encouraged to redo the whole thing but I'm not afraid of large, gradual rewrites!
